frontend/src/views/Pages/AccountOverview/GddSend/TransactionConfirmation.vue

```diff
@@ -1,44 +1,46 @@
 <template>
-  <b-row>
-    <b-col>
-      <div class="display-4 p-4">{{ $t('form.send_check') }}</div>
-      <b-list-group>
-        <b-list-group-item class="d-flex justify-content-between align-items-center">
-          {{ email }}
-          <b-badge variant="primary" pill>{{ $t('form.receiver') }}</b-badge>
-        </b-list-group-item>
-        <b-list-group-item class="d-flex justify-content-between align-items-center">
-          {{ amount }} GDD
-          <b-badge variant="primary" pill>{{ $t('form.amount') }}</b-badge>
-        </b-list-group-item>
-        <b-list-group-item class="d-flex justify-content-between align-items-center">
-          {{ memo ? memo : '-' }}
-          <b-badge variant="primary" pill>{{ $t('form.message') }}</b-badge>
-        </b-list-group-item>
-        <b-list-group-item class="d-flex justify-content-between align-items-center">
-          {{ $d($moment(date), 'long') }}
-          <b-badge variant="primary" pill>{{ $t('form.date') }}</b-badge>
-        </b-list-group-item>
-      </b-list-group>
-      <b-row>
-        <b-col>
-          <b-button @click="$emit('on-reset')">{{ $t('form.cancel') }}</b-button>
-        </b-col>
-        <b-col class="text-right">
-          <b-button variant="success" :disabled="loading" @click="$emit('send-transaction')">
-            {{ $t('form.send_now') }}
-          </b-button>
-        </b-col>
-      </b-row>
-    </b-col>
-  </b-row>
+  <div>
```

Comment:
I added a div on top level. So we can have two rows inside the div. One for the information and one for the buttons. Before it was a row nested into a row.